### PR TITLE
Adds _id and _rev back to internal datasource filter options

### PR DESCRIPTION
**Pull request by PClmnt**
## Description
The ability to select _id and _rev columns when filtering a table were missing due to the fact we now use the table schema which doesn't include these. Just adding them manually on the frontend so they can be usable again.

### CODE REVIEW

File: packages/frontend-core/src/components/FilterBuilder.svelte [hunk 1]
```diff
@@ -28,14 +28,30 @@
   export let allowBindings = false
   export let filtersLabel = "Filters"
 
+  $: {
+    if (
+      tables.find(
+        table =>
+          table._id === datasource.tableId &&
+          table.sourceId === "datasource_internal_bb_default"
```
Comment (PClmnt) [inline packages/frontend-core/src/components/FilterBuilder.svelte:36]: Just need to make this a const somewhere

File: packages/shared-core/src/constants/index.ts [hunk 2]
```diff
@@ -179,3 +179,5 @@ export enum BpmStatusValue {
   VERIFYING_EMAIL = "verifying_email",
   COMPLETED = "completed",
 }
+
+export const DEFAULT_BB_DATASOURCE_ID = "datasource_internal_bb_default"
```
Comment (mike12345567) [inline packages/shared-core/src/constants/index.ts:183]: Is this duplicated, I believe there is somewhere in backend-core or server that defines this as well?

Comment (PClmnt) [reply packages/shared-core/src/constants/index.ts:183]: Yea I wanted to add it in to frontend-core as it didn't exist there, so thought it made sense to add to shared-core for that purpose. I can't import from backend-core or server in shared-core can I?

Comment (mike12345567) [reply packages/shared-core/src/constants/index.ts:183]: You can remove it from everywhere else though and then import it from shared-core to be exported from backend-core - in `backend-core/src/constants/db.ts` you can do `export { DEFAULT_BB_DATASOURCE_ID } from "@budibase/shared-core`

Comment (PClmnt) [reply packages/shared-core/src/constants/index.ts:183]: Sorted this now @mike12345567

**Review by mike12345567 — APPROVED** [review]
LGTM!

**Pull request merged**